File: CSE331-Algorithm and Complexity/HW0/HW0Python/Driver.py
import sys
from Utility import Utility
from Solution import Solution

class Driver:

    def __init__(self):
        if len(sys.argv) < 2:
            print("Please provide the testcase filepath as a command line argument")
            return
        l = Utility().read_file(sys.argv[1])
        s = Solution(l)
        s.outputSortedList("A","B",len(l))

Driver()

File: CSE331-Algorithm and Complexity/HW0/HW0Java/src/ub/cse/algo/Driver.java
package ub.cse.algo;

import java.util.List;

/**
 * Class for running hw0. Will take in a command line argument specifying
 * the number of testcases to run.
 */
public class Driver {

    public static void main(String[] args) {
        if (args.length != 1) {
            System.out.println("Please provide the testcase filepath as the argument");
        }
        HW0Utility utility = new HW0Utility();
        List<String> l = utility.readFile(args[0]);
        Solution s = new Solution(l);
        s.outputSortedList("A","B",l.size());
    }
}
